ddcz/views.py

```diff
@@ -20,7 +20,8 @@
 from django.contrib import messages
 
 from .commonarticles import SLUG_NAME_TRANSLATION_FROM_CZ, COMMON_ARTICLES_CREATIVE_PAGES
-from .forms import LoginForm, PasswordResetForm
+from .forms.authenticationForms import LoginForm, PasswordResetForm
```

Comment:
The Python convention is not to use `camelCase` because of how import system works—please rename to `authenticationform`

Reply:
On a second look: why is `Forms` part of the module name when it's in the package? 

That would be like having `django.djangoForms.djangoFormFields`. `django.form.fields` is much nicer.